src/main/java/frc/robot/subsystems/ScoringLimelight/ScoringLimelight.java

```diff
@@ -158,15 +157,14 @@ public Optional<Transform2d> checkForTag() {
             Rotation2d.fromRadians(-cameraToTag.getRotation().getY()));
 
     System.out.println("robotToTag: " + robotToTag);
-    System.out.println("trap offset: " + ScoringLimelightCal.REEF_OFFSET);
-    System.out.println("sum: " + robotToTag.plus(ScoringLimelightCal.REEF_OFFSET));
 
-    return Optional.of(robotToTag.plus(ScoringLimelightCal.REEF_OFFSET));
+    return Optional.of(robotToTag); //don't include the offset here because it is variable and we will modify it later
```

Comment:
well, why can't we just add the offset but leave it at 0 translation, 0 rotation for now?